src/components/editTestCase/qdm/LeftPanel/ElementsTab/Elements/DataElementsCard/attributes/AttributeSection.test.tsx

```diff
@@ -485,4 +485,82 @@ describe("AttributeSection", () => {
     expect(typeOptions.length).toEqual(9);
     fireEvent.click(within(typeSelect).getByText("Decimal"));
   });
+
+  it("shows DateTime on selecting the DateTime type", async () => {
+    const assessmentElement: AssessmentPerformed = new AssessmentPerformed();
```

Comment:
I think no need to provide explicit type here. `const assessmentElement = new AssessmentPerformed();` should be fine. Typescript can infer the type from the right side assignment.